mux/test/Test/Mux.hs

```diff
@@ -0,0 +1,659 @@
+{-# LANGUAGE BangPatterns               #-}
```

Comment:
This is a rename of `ouroboros-network/test/Test/Mux.hs` so you should try and use `git mv` to preserve version history.

Reply:
It's only part of it, the `demo` stayed in `ouroboros-network`